lib/engine/game/base.rb

```diff
@@ -203,6 +203,8 @@ class Base
 
       CORPORATIONS = [].freeze
 
+      MINORS = [].freeze
```

Comment:
Is this still needed?

Reply:
I was getting an warning that is wasnt declared, but i can revert it.